pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperTestSetupUtil.java

```diff
@@ -202,15 +223,15 @@ public static void deleteKitsArray(List<String> kitIds) {
 
     }
 
-    private static String getPrimaryKey(ResultSet rs, String table) throws SQLException {
+    private String getPrimaryKey(ResultSet rs, String table) throws SQLException {
```

Comment:
This also looks pretty generic, possibly another copy paste from TestKitUtil?  See if it can be factored out to DBUtil and so there's only one version of the method.